templates/taxi/car_confirm_delete.html

```diff
@@ -1,10 +1,17 @@
 {% extends "base.html" %}
 
 {% block content %}
-  <h1>Delete car?</h1>
-  <form action="" method="post">
-    {% csrf_token %}
+  <h1>Delete car {{ car.model }}?</h1>
+    <div class="row">
+      <div class="col-1">
+        <a href="{% url 'taxi:car-detail' car.id %}" class="btn btn-secondary">Cancel</a>
```

Comment:
The 'Cancel' action is correctly implemented as a link. Ensure it is styled appropriately to look like a button using CSS classes.